pkg/virt-api/webhooks/mutating-webhook/mutating-webhook.go

```diff
@@ -123,7 +124,10 @@ func mutateVMIs(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
 	if vmi.Spec.Domain.Machine.Type == "" {
 		vmi.Spec.Domain.Machine.Type = config.GetMachineType()
 	}
-	kubev1.SetObjectDefaults_VirtualMachineInstance(&vmi)
+	v1.SetObjectDefaults_VirtualMachineInstance(&vmi)
+	// Default CPU request is done after the Resources section is initialized, if needed,
+	// in v1.SetObjectDefaults_VirtualMachineInstance. TODO: set default memory here
+	setDefaultCPURequest(&vmi, func() resource.Quantity { return config.GetCPURequest() })
```

Comment:
I don't think that you need the callback here. The config live-updates values. MutateVMI is called for each VMI individually. You should be ablee to just return `config.GetCPURequest` without `()` instead of wrapping it again.

Reply:
Right, this is a workaround for the tests - as I couldn't mock the kubevirt-config in the tests of mutating-webhook. That's the issue I referred to in #2142 - if we find a way to set values in kubevirt-config in those tests, I'll drop it

Reply:
Ah, I thought you meant different thing. ok, I'll pass the configuration value as an argument :)